R/oc_bbox.R

```diff
@@ -0,0 +1,146 @@
+#' Bounding box list for opencage queries
+#'
```

Comment:
in the title why have the word "list"?

Reply:
Because it creates a list of (sf-style) bounding box(es). Alternatively "List of bounding box(es) for opencage queries" or just "Bounding boxes for opencage queries"?